package.xml

```diff
@@ -4,10 +4,11 @@
   <version>0.0.1</version>
   <description>A Human hand model as URDF</description>
 
-  <maintainer email="gwalck@techfak.uni-bielefeld.de">Guillaume Walck</maintainer>
-  <license>NO REDISTRIBUTION ACCEPTED</license>
+  <maintainer email="rhaschke@techfak.uni-bielefeld.de">Robert Haschke</maintainer>
+  <license>GPLv2</license>
```

Comment:
Wouldn't GPL enforce you to release your source with GPL as well?

Reply:
We have no choice on this package (code derived from GPLv2 code must remain GPLv2 and cannot become LGPL) and since `human hand description` is only **loaded** on the parameter server, there is no **linking** not even as a plugin inside our software. So there is no propagation of license IMHO to the rest of the software reading the generated file of `human_hand_description` or calling the `upload.launch`.